spec/GraphQLOverHTTP.md

```diff
@@ -138,9 +140,9 @@ Note: {variables} and {extensions}, if set, must have a map as its value.
 
 ## GET
 
-For HTTP GET requests, the query parameters should be provided in the query component of the request URL in the form of
+For HTTP GET requests, the query parameters MUST be provided in the query component of the request URL in the form of
 `key=value` pairs with `&` symbol as a separator and both the key and value should have their "reserved" characters percent-encoded as specified in [section 2 of RFC3986](https://tools.ietf.org/html/rfc3986#section-2).
-The unencoded value of the {variables} parameter should be represented as a JSON-encoded string.
+The unencoded value of the {variables} parameter SHOULD be represented as a JSON-encoded string.
```

Comment:
This should probably be a `MUST`
```suggestion
The unencoded value of the {variables} parameter MUST be represented as a JSON-encoded string.
```

Reply:
(Without this, client interop is going to be an issue: what formats does the server support? What formats does the client support?)